src/panels/config/backup/ha-config-backup-details.ts

```diff
@@ -146,7 +146,7 @@ class HaConfigBackupDetails extends LitElement {
                 </ha-alert>
               `
             : !this._backup
-              ? html`<ha-circular-progress active></ha-circular-progress>`
+              ? html`<ha-spinner-delayed></ha-spinner-delayed>`
```

Comment:
Why the `delayed` here?

Reply:
I found out that `ha-circular-progress active` was a bug and didn't show any loading.
When I mirgated it to `ha-spinner` the spinner blinked shortly before the data was there, what looked odd. So I created the `delayed` component to show it after 1s of loading.

I think we would need this delayed on more places, but it's hard to find, we should replace it when we see a good use case in the software.